src/main/java/com/mastery/java/task/controllers/EmployeeController.java

```diff
@@ -0,0 +1,157 @@
+package com.mastery.java.task.controllers;
+
+import com.mastery.java.task.dto.Employee;
+import com.mastery.java.task.exceptions.DuplicateEmployeeException;
+import com.mastery.java.task.exceptions.InvalidDateException;
+import com.mastery.java.task.service.EmployeeService;
+import okhttp3.*;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.core.JmsMessagingTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.jms.Queue;
+import javax.validation.Valid;
+import java.io.IOException;
+import java.util.List;
+
+@Controller
+public class EmployeeController {
```

Comment:
please use the REST specification. If you do not specify the server.servlet.context-path, then do this separately for this controller. All the endpoints should contain 'employee' word in the path

Reply:
i had servlet.context-path in properties, and I will add "employee" to all endpoints